examples/src/components/GeoPushpinChartConfigurationPointsExample.jsx

```diff
@@ -0,0 +1,149 @@
+            togglePinSizes: false,
+        };
+    }
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onError", ...params);
+    }
+
+    onZoomChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onZoomChanged", ...params);
+    }
+
+    onCenterPositionChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onCenterPositionChanged", ...params);
+    }
+
+    onPointSizeChange = event => {
+        const { id, value } = event.target;
+        this.setState(prevState => {
+            return {
+                ...prevState,
+                points: {
+                    ...prevState.points,
+                    [id]: value,
+                },
+            };
+        });
+    };
+
+    togglePinSizes = () => {
+        this.setState(prevState => {
+            return {
+                ...prevState,
+                togglePinSizes: !prevState.togglePinSizes,
+            };
+        });
+    };
+
+    toggleGroupNearbyPoints = () => {
+        this.setState(prevState => {
+            return {
+                ...prevState,
+                points: {
+                    ...prevState.points,
+                    groupNearbyPoints: !prevState.points.groupNearbyPoints,
+                },
+                togglePinSizes: false,
+            };
+        });
+    };
+
+    renderPointSizeDropDown = id => {
+        return (
+            <span style={{ display: "inline-block", minWidth: "10em", verticalAlign: "middle" }}>
+                {id}
+                <select id={id} onChange={this.onPointSizeChange}>
+                    {POINT_SIZE_OPTION.map((size, index) => (
+                        <option key={id + index.toString()} value={size.value}>
+                            {size.label}
+                        </option>
+                    ))}
+                    ;
+                </select>
+            </span>
+        );
+    };
+
+    render() {
+        const style = { height: "500px" };
+        const { points, togglePinSizes } = this.state;
+        const geoConfig = {
+            mapboxToken: MAPBOX_TOKEN,
+            points,
+        };
+
+        return (
+            <div className="s-geo-chart">
+                <style jsx>
+                    {`
+                        .hidden {
+                            display: none;
+                        }
+                    `}
+                </style>
+                <button className="s-change-points-size" onClick={this.togglePinSizes}>
+                    Custom Points Size
+                </button>
```

Comment:
should we create 2 examples: 1 for clustering points, 1 for sizing points?
This could make examples easier to read
c.c @naibinh @dlcd2203

Reply:
I couldn't agree more :)

Reply:
fixed